solution/operator/crds/kafka_operator_crd.yaml

```diff
@@ -0,0 +1,2472 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Should be put under `solution/operator/deploy/crds/`.

Reply:
This file was removed entirely